### PR TITLE
Add support for group tags.

### PR DESCRIPTION
**Pull request by royelbaum**
Change the tags file to be more generic.
Fix tags test add check equals sizes.

### CODE REVIEW

**Review by mendi160 — APPROVED** [review]

File: athenz/resource_group.go [hunk 1]
```diff
@@ -190,6 +215,21 @@ func resourceGroupUpdate(ctx context.Context, d *schema.ResourceData, meta inter
...
 	}
+
+	if d.HasChange("tags") {
+		isGroupChange = true
+		_, n := d.GetChange("tags")
+		tags := expandTagsMap(n.(map[string]interface{}))
+		group.Tags = tags
+	}
+
+	if isGroupChange {
```
Comment (dvirguttman) [inline athenz/resource_group.go:226]: why do we set this flag only when tag changed and not in members? I am confused
or updateGroupMembers take care of that and then the parameter name should be isGroupTagsChanged or something like that

**Review by mendi160 — COMMENTED** [review]
general note: in resourceGroupUpdate since you create the groupMeta object locally, I'm pretty sure you will override all the meta fields that are unsupported by terraform (e.g. selfServe). if this behavior is not desired, instead of creating the groupMeta object locally you should retrieve it and modify the relevant fields

**Review by mendi160 — COMMENTED** [review]
You should add tags to the data_source_group's schema too.
Also, please remember to update the doc files.

**Review by mendi160 — COMMENTED** [review]
@royelbaum please update the file `docs/data-sources/athenz_group.md` as well

**Review by mendi160 — APPROVED** [review]

**Review by dvirguttman — APPROVED** [review]

**Pull request merged**